app/models/user.rb

```diff
@@ -229,18 +229,38 @@ def admin?
     self.title == 'Staff' || self.title == 'Mod'
   end
 
-  def feed
-    @feed ||= Feed.user(id)
+  def posts_feed
+    @posts_feed ||= Feed.user_posts(id)
+  end
+
+  def media_feed
+    @media_feed ||= Feed.user_media(id)
   end
 
   def aggregated_feed
     @aggr_feed ||= Feed.user_aggr(id)
   end
 
+  def posts_aggregated_feed
+    @posts_aggr_feed ||= Feed.user_posts_aggr(id)
+  end
+
+  def media_aggregated_feed
+    @media_aggr_feed ||= Feed.user_media_aggr(id)
+  end
+
   def timeline
     @timeline ||= Feed.timeline(id)
   end
 
+  def posts_timeline
+    @posts_timeline ||= Feed.timeline_posts(id)
+  end
+
+  def media_timeline
+    @media_timeline ||= Feed.timeline_media(id)
+  end
+
```

Comment:
This list of methods is getting to some silly levels, perhaps an abstraction would help.  Maybe moving the filter-to-feed-name conversion down to the `Feed` layer, so we can do `Feed.timeline(id, filter: :media)` or something?  Would probably help clarify the logic elsewhere too.

Alternatively, and this is a bit crazy, the `Feed` object could be changed to be a "virtual feed" representing a set of filter feeds so that it still _feels_ like interacting with unsplit feeds, but under the hood it's actually using multiple feeds